automatic/GoogleChrome/update.ps1

```diff
@@ -1,6 +1,8 @@
 import-module au
+. "$PSScriptRoot\..\..\scripts\Get-Padded-Version.ps1"
 
 $releases = 'http://omahaproxy.appspot.com/all?os=win&amp;channel=stable'
+$paddedUnderVersion = '56.0.2925'
```

Comment:
I like staying on the side of safety for now. If we need to do this later it is easy enough to add back in.